userspace/libscap/engine/gvisor/gvisor_public.h

```diff
@@ -30,6 +30,7 @@ extern "C"
 
 		bool no_events; //< Pinky swear we don't want any event from it (i.e. next will always fail, just have proc scan)
 		int gvisor_epoll_timeout;	///< When using gvisor, the timeout to wait for a new event
+		struct scap_platform *gvisor_platform; ///< The gvisor engine and platform have a bit of shared state
```

Comment:
IMO this needs to be a `scap_gvisor_platform` since we actually depend on the concrete type. Forward-declare the struct type if needed.

Reply:
Makes sense, done!